src/options.js

```diff
@@ -1,84 +1,78 @@
 // Based on https://developer.chrome.com/extensions/optionsV2
 
+const flashMessage = function(message) {
```

Comment:
Here and in other places: can you please use arrow functions for consistency?

Reply:
I can where possible. Does this applies to `extension.js` also? As arrow functions does not get their own scope using them is not always possible (e.g. constructors).

Reply:
Yes, please use the method shorthand when defining object methods  (https://google.github.io/styleguide/jsguide.html#features-objects-method-shorthand), and in other cases like here arrow functions with const.